apiary.apib

```diff
@@ -5797,3 +5797,51 @@ Development branches allow you to prepare changes to configuration and tables in
                     "outBytesUncompressed": 0
                 }
             }
+
+## Branch manipulation [/v2/storage/dev-branches/{id}]
+
+### Delete branch [DELETE]
+
++ Parameters
+    + id (int) - id of the branch
+
++ Request (application/x-www-form-urlencoded)
```

Comment:
Proč je potřeba tento typ, když je to `DELETE` a neposílám v requestu žádná data?
@tomasfejfar

Reply:
Formálně to tenhle typ je. 

Za čas bude třeba možnost force u mazání - že to smaže jen pokud tam nejsou žádné změny nebo tak něco. A tam by ten parametr měl být taky takhle... `force=1` v body. Tak proto ta hlavička.

Reply:
Ok díky za vysvětlení.